src/conf.ml

```diff
@@ -27,10 +27,21 @@ let auth =
     None
   )
 
-let pool_name os_family arch =
+let pool_name (distro:Dockerfile_distro.t) arch =
+  let os_family = Dockerfile_distro.os_family_of_distro distro in
   let os_str = match os_family with
-  | `Windows | `Cygwin -> "windows"
-  | `Linux -> "linux" in
+  | `Windows | `Cygwin ->
+    let dedicated_pool = [`V1809; `V2004; `V20H2; `Ltsc2022] in
+    begin match Dockerfile_distro.resolve_alias distro with
+    | `Windows (_, release) | `Cygwin release when List.mem release dedicated_pool ->
+      (* Temporary but convenient alias until Microsoft releases 21H1 images. *)
```

Comment:
From the announcements, I think the point is that Microsoft is _not_ going to release 21H1 images - it's tracking Windows Server specifically. The fate of the 6-monthly SAC channels isn't totally clear (there's some Azure Kubernetes stuff, but it's unclear whether this will diverge at this early stage where the releases are identical)